### PR TITLE
Change device type description

### LINKED ISSUE #1549: Explain `Change device type` feature
It would be good to surface what this feature means, how it is done and who has permissions to do it. I'd expect to see this feature listed at least on this page: https://www.balena.io/docs/learn/manage/actions/#device-specific-actions

### PR DESCRIPTION
**Pull request by konmouz**
Added description of the Change device type action, closes #1549 

Change-type: patch
Signed-off-by: konmouz <konstantinos@balena.io>

### CODE REVIEW

File: pages/learn/manage/actions.md [hunk 1]
```diff
@@ -100,6 +100,16 @@ The `Delete Device` action is an extremely dangerous action and results in disas
 
 This action allows you to [enable support access][support-access] to an individual devices for a set time period.
 
+### Change device type
+
+If one or more devices has been added to an application with the wrong [device type](https://www.balena.io/docs/reference/hardware/versioning/#device-types), you have the ability to change it on the dashboard.
+
+Option 1: On the application’s device list, select one or more devices and clip the `Actions` drop down on the top right corner of the list. Select `Change device type` and follow the instructions on the modal.
```
Comment (vipulgupta2048) [inline pages/learn/manage/actions.md:107]:
```suggestion
Option 1: On the application’s device list, select one or more devices and click the `Actions` drop-down menu on the top right corner. Select `Change device type` option from the list and follow the instructions on the modal.
```

File: pages/learn/manage/actions.md [hunk 2]
```diff
@@ -100,6 +100,16 @@ The `Delete Device` action is an extremely dangerous action and results in disas
 
 This action allows you to [enable support access][support-access] to an individual devices for a set time period.
 
+### Change device type
+
+If one or more devices has been added to an application with the wrong [device type](https://www.balena.io/docs/reference/hardware/versioning/#device-types), you have the ability to change it on the dashboard.
```
Comment (vipulgupta2048) [inline pages/learn/manage/actions.md:105]:
```suggestion
If one or more devices has been added to an application with the wrong [device type](https://www.balena.io/docs/reference/hardware/versioning/#device-types), one can change the device-type for their devices through the dashboard. There are 2 ways to go about this, 
```

File: pages/learn/manage/actions.md [hunk 3]
```diff
@@ -100,6 +100,16 @@ The `Delete Device` action is an extremely dangerous action and results in disas
...
+
+Option 1: On the application’s device list, select one or more devices and clip the `Actions` drop down on the top right corner of the list. Select `Change device type` and follow the instructions on the modal.
+
+Option 2: On the device page, click `Actions` on the left-hand side menu, scroll down to `Dangerous actions`, click `Change device type` and follow the instructions on the modal.
```
Comment (vipulgupta2048) [inline pages/learn/manage/actions.md:109]:
```suggestion
Option 2: On the device page, click `Actions` tab on the left sidebar menu. Scroll down to the `Dangerous actions` section and click `Change device type` after which follow the instructions on the modal to change your device-type.
```

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by konmouz** [comment]
@balena-ci I self-certify!

**Pull request merged**